src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -91,3 +92,75 @@ def cluster_projects(projects: Sequence[Project]) -> None:
             metrics.incr(
                 "txcluster.cluster_projects.unclustered", amount=unclustered, sample_rate=1.0
             )
+
+
+@instrumented_task(
+    name="sentry.ingest.span_clusterer.tasks.spawn_span_cluster_projects",
+    queue="span.descs.clusterer",  # XXX(iker): we should use a different queue
+    default_retry_delay=5,
+    max_retries=5,
+    soft_time_limit=PROJECTS_PER_TASK * CLUSTERING_TIMEOUT_PER_PROJECT,
+    time_limit=PROJECTS_PER_TASK * CLUSTERING_TIMEOUT_PER_PROJECT + 2,
+)
+def spawn_span_cluster_projects(**kwargs: Any) -> None:
+    """Look for existing span description sets in redis and spawn clusterers for each"""
+    with sentry_sdk.start_span(op="span_descs-cluster_spawn"):
+        project_count = 0
+        project_iter = redis.get_active_projects(ClustererNamespace.SPANS)
+        project_iter = filter(
+            lambda p: features.has("projects:span-metrics-extraction", p), project_iter
+        )
```

Comment:
Using `filter` (vs a list comprehension) because we're interested in having an iterator for the `islice` below.

Reply:
Nice! TIL